pkgs/tools/text/miller/default.nix

```diff
@@ -1,15 +1,13 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, flex, libtool }:
+{ stdenv, fetchurl, autoreconfHook, flex, libtool }:
```

Comment:
Why are you switching away from `fetchFromGitHub`? I see the release tag got changed to have a "v" at the beginning. You can address this with `rev = "v${version};".

Reply:
You are right. Fixed. Thanks for spotting this.